studio/components/grid/components/editor/JsonEditor.tsx

```diff
@@ -92,30 +92,34 @@ export function JsonEditor<TRow, TSummaryRow = unknown>({
                 <p className="text-xs text-scale-1100">Cancel changes</p>
               </div>
             </div>
-            <Tooltip.Root delayDuration={0}>
-              <Tooltip.Trigger>
-                <div
-                  className={[
-                    'border border-scale-700 rounded p-1 flex items-center justify-center',
-                    'transition cursor-pointer bg-scale-600 hover:bg-scale-700',
-                  ].join(' ')}
-                  onClick={() => onSelectExpand()}
-                >
-                  <IconMaximize size={12} strokeWidth={2} />
-                </div>
-              </Tooltip.Trigger>
-              <Tooltip.Content side="bottom">
-                <Tooltip.Arrow className="radix-tooltip-arrow" />
-                <div
-                  className={[
-                    'rounded bg-scale-100 py-1 px-2 leading-none shadow',
-                    'border border-scale-200',
-                  ].join(' ')}
-                >
-                  <span className="text-xs text-scale-1200">Expand editor</span>
-                </div>
-              </Tooltip.Content>
-            </Tooltip.Root>
+            <Tooltip.Provider>
```

Comment:
we shouldn't be calling Tooltip.Provider at every level - there should only be one Provider that's called at the app level ([ref](https://www.radix-ui.com/docs/primitives/components/tooltip#provider))